pyproject.toml

```diff
@@ -14,8 +14,9 @@ version = "1.4.3"
 "Changelog" = "https://github.com/niksite/url-normalize#url-normalize"
 
 [tool.poetry.dependencies]
-python = "~2.7 || ^3.6"
+python = "^3.6"
 six = "*"
```

Comment:
If python 2.7 support is dropped, `six` (and [its usage in `tools.py`](https://github.com/niksite/url-normalize/blob/6c0be91584de586a150c57026d516a0633eb22ed/url_normalize/tools.py#L6-L9)) could also be removed.